tests/rbd/test_rbd_image_migration_qcow.py

```diff
@@ -0,0 +1,191 @@
+        kw: Key/value pairs of configuration information to be used in the test
+    """
+    qcow_spec = {
+        "type": "qcow",
+        "stream": {
+            "type": "http",
+            "url": "http://download.ceph.com/qa/ubuntu-12.04.qcow2",
+        },
+    }
+
+    for pool_type in dest_obj.get("pool_types"):
+        rbd_config = kw.get("config", {}).get(pool_type, {})
+        multi_pool_config = getdict(rbd_config)
+
+        for pool, pool_config in multi_pool_config.items():
+            image_spec = pool + "/" + random_string(len=5)
+            # prepare migration
+            if migration_prepare(
+                src_spec=qcow_spec,
+                dest_spec=image_spec,
+                client_node=client,
+                **kw,
+            ):
+                raise Exception("Failed to prepare migration")
+            else:
+                log.info("Migration prepared successfully")
+
+            # verify prepare migration status
+            if verify_migration_state(
+                action="prepare",
+                dest_spec=image_spec,
+                client_node=client,
+                **kw,
+            ):
+                raise Exception("Failed to prepare migration")
+                return 1
+            else:
+                log.info("Migration prepare status verfied successfully")
+
+            # execute migration
+            if migration_action(
+                action="execute",
+                dest_spec=image_spec,
+                client_node=client,
+                **kw,
+            ):
+                raise Exception("Failed to execute migration")
+                return 1
+            else:
+                log.info("Migration executed successfully")
+
+            # verify execute migration status
+            if verify_migration_state(
+                action="execute",
+                dest_spec=image_spec,
+                client_node=client,
+                **kw,
+            ):
+                raise Exception("Failed to execute migration")
+                return 1
+            else:
+                log.info("Migration executed successfully")
+
+            # commit migration
+            if migration_action(
+                action="commit",
+                dest_spec=image_spec,
+                client_node=client,
+                **kw,
+            ):
+                raise Exception("Failed to commit migration")
+                return 1
+            else:
+                log.info("Migration committed successfully")
+
+            # verify commit migration status
+            if verify_migration_state(
+                action="commit",
+                dest_spec=image_spec,
+                client_node=client,
+                **kw,
+            ):
+                raise Exception("Failed to commit migration")
+            else:
+                log.info("Migration committed successfully")
+
+            out = get_disk_usage_for_image(client, image_spec)
```

Comment:
Are you writing any IOs before migration? If not, I strongly suggest you to run IOs because for any test case running IOs is very important. 
Also post migration perform data integrity verification using md5sum.

Reply:
Done.